binding/python/__init__.py

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright 2022-2023 Picovoice Inc.
```

Comment:
we generally only update the copyright notice only when we make a change the file